com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -182,41 +209,38 @@ private void Sort()
         public string ToBase64()
         {
             NetworkConfig config = this;
-            using (PooledNetworkBuffer buffer = PooledNetworkBuffer.Get())
+            using (var buffer = PooledNetworkBuffer.Get())
```

Comment:
Huh, cool, didn't think of this but makes sense.  I read this might [generate a warning ](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2015/code-quality/ca2202-do-not-dispose-objects-multiple-times?view=vs-2015&redirectedfrom=MSDN)

Reply:
I think we're not double-disposing here 👀

Reply:
`NetworkWriter` will not cause `NetworkBuffer` disposal, if that's what we're concerned about.